src/Symfony/Component/Filesystem/Filesystem.php

```diff
@@ -525,13 +525,21 @@ public function isAbsolutePath($file)
      * @param null|int $mode     The file mode (octal). If null, file permissions are not modified
      *                           Deprecated since version 2.3.12, to be removed in 3.0.
      *
-     * @throws IOException If the file cannot be written to.
+     * @throws IOException if the file cannot be written to
```

Comment:
I think we disabled the fabbot rule triggering this, can you check?

Reply:
No, I've fixed everything in the codebase... using php-cs-fixer.